src/sentry/integrations/github/integration.py

```diff
@@ -79,6 +80,21 @@ def get_repositories(self):
             data.append({'name': repo['name'], 'identifier': repo['full_name']})
         return data
 
+    def search_issues(self, query):
```

Comment:
Is `search_issues` the right name for this method? Seems like it can search more than just that